docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -608,11 +626,12 @@ OIDC `CodeAuthenticationMechanism` uses the default `io.quarkus.oidc.TokenStateM
 
 It makes Quarkus OIDC endpoints completely stateless and it is recommended to follow this strategy to achieve the best scalability results.
 
-See <<db-token-state-manager>> and <<custom-token-state-manager>> sections of this guide for alternative approaches where tokens can be stored in the database or other server-side storage, if you prefer and have good reasons for storing the token state on the server.
+See the <<db-token-state-manager>> and <<custom-token-state-manager>> sections of this guide for alternative approaches to storing tokens.
+For example, storing tokens in the database or other server-side storage, if you prefer and have good reasons for storing the token state on the server.
 
-You can configure the default `TokenStateManager` to avoid saving an access token in the session cookie and only keep ID and refresh tokens or ID token only.
+You can configure the default `TokenStateManager` to avoid saving an access token in the session cookie and to only keep ID and refresh tokens or ID tokens only.
```

Comment:
```suggestion
You can configure the default `TokenStateManager` to avoid saving an access token in the session cookie and to only keep ID and refresh tokens or ID token only.
```

session cookie can keep a single ID token

Reply:
Thanks for clarifying. For clarity, I updated this to say 
"session cookie and to only keep ID and refresh tokens or a single ID token only."